### PR TITLE
PR for issue 998 - only missing new video content

### PR DESCRIPTION
**Pull request by Bangalisch**
Just needs new video content.
Moved Testnet away from this page as a avg user will not use it. 
It was moved to Deployment, it could use a little touching up with Screenshots.

### CODE REVIEW

**Comment by Bangalisch** [comment]
#998

**Review by pavlenex — CHANGES_REQUESTED** [review]
Nice idea, agreed with it, but I think `Developers` is better suitable place for this then deployment. Dev are more likely to use testnet to test things out then people trying to find docs on how to actually deploy btcpay, though the line is thin there, I suggest we add it in `Developers` docs, specifically here:

Screenshot 2022-03-30 at 11 06 11

What do you say?

**Comment by Bangalisch** [comment]
Agreed! 
Wasnt to sure about the Testnet stuff where to put it, but ur right! 
Lets move it there

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -23,10 +23,10 @@ const sidebarUserGuide = [
     collapsable: false,
     children: [
       ['/Guide', 'Introduction'],
+      ['/TryItOut', 'Try it out']
```
Comment (dennisreimann) [inline docs/.vuepress/config.js:26]:
```suggestion
      ['/TryItOut', 'Try it out'],
```

**Pull request merged**